server/controllers/product.js

```diff
@@ -1,36 +1,24 @@
 import slugify from 'slugify';
 import fs from 'fs';
 import Product from '../models/product.js';
+import { validateCreate } from '../validators/product/productValidation.js';
 
 export const create = async (req, res) => {
   try {
     const { name, description, price, category, quantity, shipping } = req.fields;
     const { photo } = req.files;
-
-    switch (true) {
-      case !name.trim():
-        res.json({ error: 'Name is required' });
-      case !description.trim():
-        res.json({ error: 'description is required' });
-      case !price.trim():
-        res.json({ error: 'Price is required' });
-      case !category.trim():
-        res.json({ error: 'Category is required' });
-      case !quantity.trim():
-        res.json({ error: 'Quantity is required' });
-      case !shipping.trim():
-        res.json({ error: 'Shipping is required' });
-      case photo && photo.size > 100000:
-        res.json({ error: 'Shipping is required' });
+    console.log("aqui")
+    const error = await validateCreate({name,description,price,category,quantity,shipping,photo})
```

Comment:
Faltando a validacao do eslint 

```suggestion
    const error = await validateCreate({name, description, price, category, quantity, shipping, photo})
```